katti/unsolved_problems.json

```diff
@@ -0,0 +1 @@
+{"carrots": 1.4}
```

Comment:
This should probably just be an empty json string `{}` for the purpose of the commit

Reply:
i was getting errors when it was an empty json string. I'll look into it